### DISCUSSION TITLE
Update to German locale [de]

### DISCUSSION THREAD

**Discussion by container1** [aclap-dev/video-downloadhelper#846] [Language & Translation]
Hello
I tried to translate and support as requested in explanation.
Unfortunately I noticed 4 strings are mentioned but onl3 strings where found.
[vdh-settings[de].json](https://github.com/user-attachments/files/15751195/vdh-settings.de.json)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
